topology/pynest/hl_api.py

```diff
@@ -2087,6 +2087,7 @@ def PlotKernel(ax, src_nrn, mask, kern=None, mask_color='red',
     import matplotlib
     import matplotlib.pyplot as plt
     import numpy as np
+    from math import cos, sin, pi
```

Comment:
Importing "naked" functions is not a good idea, in particular since we also have `numpy` here. I would use `import math` and then `math.cos()` etc below.